### ISSUE TITLE
Can't sign in the Angular 9 sample App with guard on root path

### ISSUE THREAD

**Issue by graphicsxp** [AzureAD/microsoft-authentication-library-for-js#1414]
## Library
"@azure/msal-angular": "^1.0.0-beta.3",

## Framework
Angular 9

## Description
I tried the sample for Angular 9 and it works fine with my Azure AD. But I added the MsalGuard to the root path : 

    ```
     const routes: Routes = [
     {
       path: 'profile',
       component: ProfileComponent,
       canActivate: [
         MsalGuard
      ]
     },
     {
       path: '',
       component: HomeComponent,
       canActivate: [
         MsalGuard
      ]
     }
   ];


When I launch the application, the sign-in popup shows up immediately, I enter my credentials and the popup disappears but I'm not signed in. In azure AD I've checked the sign-in logs and it says : 

Status	
Interrupted
Sign-in error code	
50058
Failure reason	
The application tried to perform a silent sign in and the user could not be silently signed in. The application needs to start an interactive flow giving users an option to sign in. Contact app owner.

## Expected behavior
I should be signed-in.

**Comment by jasonnutter** [comment]
@graphicsxp The page used as a redirect URI should not itself require authentication. The MSAL Guard should now catch this scenario and prevent itself from redirecting inside the iframe. This is available in `@azure/msal-angular@1.0.0-beta.4`, can you try this version?

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/changelog.md#100-beta4

As a best practice, we recommend redirect pages not themselves require authentication. This can be done by setting the redirect uri to a blank html page (that has no content at all). I'll be updating the Angular samples to show this pattern.

**Comment by jasonnutter** [comment]
Closing due to inactivity, let us know if you have further questions.
